echopype/clean/api.py

```diff
@@ -40,14 +44,20 @@ def mask_transient_noise(
         Calibrated Sv data with depth data variable.
     func: str, default `nanmean`
         Pooling function used in the pooled Sv aggregation.
-    depth_bin : str, default `20`m
-        Pooling radius along ``depth`` in meters.
-    num_side_pings : int, default `50`
+    depth_bin : str, default `10m`
```

Comment:
This is one example, but could you change all the things attached with a unit to be in double quotes? I think backtick is not the right format to be rendered in the docstring?
```suggestion
    depth_bin : str, default "10m"
```

Reply:
On my end, I see the following when using "10m":
![image](https://github.com/OSOceanAcoustics/echopype/assets/87830138/e26363ee-4d2f-4fe4-9e64-5e929ef7d502)

When using `10m` (with a backtick on either side), I see the following:
![image](https://github.com/OSOceanAcoustics/echopype/assets/87830138/af16516d-92e7-43e8-b4fa-e7b226b94f9f)

And this is what the read the docs shows (with backticks):
![image](https://github.com/OSOceanAcoustics/echopype/assets/87830138/857d6e13-c48c-4c22-8187-3882a9d92a70)

Reply:
Ok I think it is weird that nanmean is shown with backticks but 10m is not shown with backticks in the readthedocs.

Reply:
Ok yeah I think I agree now that this should be double quotes instead of backticks. I'll make this change.